src/Symfony/Component/Form/ResolvedFormType.php

```diff
@@ -56,9 +56,7 @@ public function __construct(FormTypeInterface $innerType, array $typeExtensions
     }
 
     /**
-     * Returns the prefix of the template block name for this type.
-     *
-     * @return string The prefix of the template block name
+     * {@inheritdoc}
```

Comment:
this is wrong as ResolvedFormTypeInterface does not define a getBlockPrefix. So either the method must be added to that interface as well. Or the public method should be removed from ResolvedFormType. I don't see why we need that public method on the resolved type as one can access it easily through `getInnerType`